core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7394,6 +7394,14 @@ public String toString() {
           .setScope(Scope.WORKER)
           .build();
 
+  public static final PropertyKey DORA_WORKER_META_STORE_ROCKSDB_DIR =
```

Comment:
METASTORE is a single word, do not need to separate them

Reply:
OK

Reply:
Fixed